src/stdlib_math.fypp

```diff
@@ -292,4 +298,27 @@ contains
     end function clip_${k1}$
 
     #:endfor
+
+    #:for k1, t1 in INT_KINDS_TYPES
+    !> Determine the greatest common divisor of two integers
+    !> ([Specification](../page/specs/stdlib_math.html#gcd))
+    !>
+    !> Version: experimental
```

Comment:
This part needs to be moved up at `gcd` interface. You can write any comments here specific to this particular procedure like:
```suggestion
    !> Returns greatest common divisor of two integers of kind ${k1}$
```

Reply:
I switched back and forth between "returns" and "determines", it's not uniform among the existing procedures as far as I remember.

Is there some guide about what goes where? I did not know if the doc scripts would recognize annotations to the interface declarations.

Reply:
With "returns" it get's clear that this procedure is expected to return a value.